sky/sdk/example/game/lib/sprite_box.dart

```diff
@@ -150,6 +154,8 @@ class SpriteBox extends RenderBox {
   }
 
   void handleEvent(Event event, _SpriteBoxHitTestEntry entry) {
+    if (!attached) return;
```

Comment:
Please put if statement bodies, especially control flow statements like 'return', clearly on their own line so that they are obvious when scanning the code.